driver/src/main/java/org/neo4j/driver/internal/async/pool/ConnectionPoolImpl.java

```diff
@@ -159,6 +160,7 @@ public int idleConnections( BoltServerAddress address )
                     ChannelPool pool = entry.getValue();
 
                     log.info( "Closing connection pool towards %s", address );
+                    nettyChannelTracker.destructAllChannels();
```

Comment:
We have a single channel tracker for all pools. This method is invoked once for every pool. Shouldn't it be called once outside of the outer loop?